### PR TITLE
Alpha 0.1.7/341 add list of recently opened log files

### PR DESCRIPTION
**Pull request by Kwash67**

### CODE REVIEW

File: gcs/electron/preload.js [hunk 1]
```diff
@@ -5,7 +5,9 @@ import path from 'node:path';
 contextBridge.exposeInMainWorld('ipcRenderer', {
   ...withPrototype(ipcRenderer),
   loadFile: (data) => ipcRenderer.invoke('fla:open-file', data),
-  getFgcsLogs: () => ipcRenderer.invoke('fla:get-fgcs-logs'),
+  // getFgcsLogs: () => ipcRenderer.invoke('fla:get-fgcs-logs'),
```
Comment (NexInfinite) [inline gcs/electron/preload.js:8]: is this still needed?

File: gcs/src/fla.jsx [hunk 2]
```diff
@@ -238,12 +238,15 @@ export default function FLA() {
         setLoadingFile(false)
       }
     }
+    if(file == null){
```
Comment (NexInfinite) [inline gcs/src/fla.jsx:241]: preferably, the code style should go `if (file == null) {` with the spaces

File: gcs/electron/main.ts [hunk 3]
```diff
@@ -256,34 +254,34 @@ app.on('activate', () => {
 
 app.whenReady().then(() => {
   createLoadingWindow()
+  // Open file
   ipcMain.handle('fla:open-file', openFile)
-  ipcMain.handle('fla:get-fgcs-logs', async () => {
-    const fgcsLogsPath = path.join(os.homedir(), 'FGCS', 'logs')
+  // Get Recent Logs
```
Comment (NexInfinite) [inline gcs/electron/main.ts:259]: these comments can be combined into one `// Open file and get recent logs` would look a little cleaner

**Review by NexInfinite — CHANGES_REQUESTED** [review]
I'm nitpicking just a bit here, code is all good but some styles would make the file look a bit more consistent.

File: gcs/electron/preload.js (hunk 1 above)
Comment (Kwash67) [reply gcs/electron/preload.js:8]: nah, i'll take it out

File: gcs/src/fla.jsx (hunk 2 above)
Comment (Kwash67) [reply gcs/src/fla.jsx:241]: okay

**Review by NexInfinite — APPROVED** [review]
LGTM

**Pull request merged**